src/layouts/components/Homepage/AnnouncementBody.tsx

```diff
@@ -0,0 +1,240 @@
+            <Editable.EmptySection
+              isEmpty={announcementItems.length === 0}
+              title="Announcements you add will appear here"
+              subtitle=""
+            >
+              <Editable.Section px={0} spacing="0.75rem" py="1.5rem">
+                {announcementItems.map(
+                  (
+                    {
+                      title: announcementTitle,
+                      date: announcementDate,
+                      announcement: announcementContent,
+                      link_text: announcementLinkText,
+                      link_url: announcementLinkUrl,
+                    },
+                    announcementIndex
+                  ) => {
+                    return (
+                      <Editable.DraggableAccordionItem
+                        title={announcementTitle || "New announcement"}
+                        draggableId={`announcement-${announcementIndex}-draggable`}
+                        index={announcementIndex}
+                        isInvalid={_.some(
+                          errors.announcementItems[announcementIndex]
+                        )}
+                        isNested
+                      >
+                        <Editable.Section>
+                          <FormControl
+                            isInvalid={
+                              !!errors.announcementItems[announcementIndex]
+                                .title
+                            }
+                            isRequired
+                          >
+                            <FormLabel>Title</FormLabel>
+                            <Input
+                              placeholder="Announcement title"
+                              id={`announcement-${announcementIndex}-title`}
+                              value={announcementTitle}
+                              onChange={onChange}
+                            />
+                            <FormErrorMessage>
+                              {
+                                errors.announcementItems[announcementIndex]
+                                  .title
+                              }
+                            </FormErrorMessage>
+                          </FormControl>
+                          <FormControl
+                            isInvalid={
+                              !!errors.announcementItems[announcementIndex].date
+                            }
+                            isRequired
+                          >
+                            <FormLabel>Date</FormLabel>
+                            <DatePicker
+                              id={`announcements-${announcementIndex}-date`}
+                              inputValue={toCmsPanelDateFormat(
+                                announcementDate
+                              )}
+                              onInputValueChange={(value) => {
+                                console.log({ value })
```

Comment:
could we also remove this?

Reply:
4883ebf13a13fc7a253afb1dd7634a9649f2c32c